client/js/plugins/sails.js

```diff
@@ -6,6 +6,11 @@ const { isProd } = require('../../../utils/config-utils');
 
 export const io = require('sails.io.js')(require('socket.io-client'));
 
+export const reconnectSockets = () => {
+  io.socket.disconnect();
+  io.socket.reconnect();
+};
```

Comment:
We need this to refresh any session connections in progress to make sure they get the new session id when the user logs in or out and the cookie changes.